### ISSUE TITLE
Proposed - TimeControl descriptor for delay

### ISSUE THREAD

**Issue by JeNeSuisPasDave** [fsmosca/PGN-Standard#13]
Section 9.6.1 describes then content of the TimeControl tag. It lists 6 different formats for describing the time control of a period. None of the six cover the common specification of a delay.  The closest one can get is to use the notation for an increment.

I propose adding a 7th format type to cover delay. It would be identical to the text for increment, except that instead of a '+' character preceding the increment seconds, a 'd' character would precede the delay seconds.

Specifically this text:

The seventh TimeControl field kind is used for a "delay" control period.
It should only be used for the last descriptor in a TimeControl tag value and
is usually the only descriptor in the value. The format consists of two
positive integers separated by a lowercase letter D ("d") character. The first integer
gives the minimum number of seconds allocated for the period and the second
integer gives the number of extra seconds added after each move is made. So,
an delay time control of 45 minutes with a pause of 5 seconds preceding
each move would be given by "2700d5" in the TimeControl tag value.

**Comment by fsmosca** [comment]
Was there a tournament that uses a delay TC? If so how they indicated this in a PGN?

I tried searching but could not find it.

**Comment by JeNeSuisPasDave** [comment]
Many tournaments use delay seconds in their time controls, especially scholastic tournaments or any tournament where there is no luxury of allowing games to continue indefinitely. My own club uses G45/D5 (game in 45 minutes with a delay of 5 seconds per move) because the games start at 7PM and the venue where we play closes at 9PM. These tournaments tend to not collect or publish game records, so I don't have example of how people have worked around the gap in the PGN specification.

However, I found this Stack Exchange interaction which suggests the only way to record the delay time and be compliant with the PGN standard is add it in an end-of-line comment after the the TimeControl tag.  

https://chess.stackexchange.com/questions/37493/bronstein-timing-in-dgt-clocks-and-the-related-pgn

The problem with an end-of-line comment is that no PGN reader/viewer is going to see it or process it as meaningful text.

So I am proposing that the standard explicitly support a 'delay' (aka Bronstein timing) descriptor within the value of the TimeControl tag.

**Comment by fsmosca** [comment]
> The seventh TimeControl field kind is used for a "delay" control period.
> It should only be used for the last descriptor in a TimeControl tag value and
> is usually the only descriptor in the value. The format consists of two
> positive integers separated by a lowercase letter D ("d") character. The first integer
> gives the minimum number of seconds allocated for the period and the second
> integer gives the number of extra seconds added after each move is made. So,
> an delay time control of 45 minutes with a pause of 5 seconds preceding
> each move would be given by "2700d5" in the TimeControl tag value.

Can you make a pull request?

From above:

```
The first integer gives the minimum number of seconds allocated for the period and the second
integer gives the number of extra seconds added after each move is made.
```

I would like to clarify on:

```
and the second integer gives the number of extra seconds added after each move is made.
```

So what is the digital clock mechanism for TC with delay? If delay is 5s and player made a move in 2s, it is obvious that 3s will not be added on the base time.

**Comment by JeNeSuisPasDave** [comment]
Yes, I can make a pull request. I'll do it this weekend (when I'm not tired and have time to be careful :-)).

The clock mechanism is that the clock doesn't start counting down until the delay has been exceeded. So if your clock was at 03:19 (meaning you had 3 minutes 19 seconds left on your clock) and it became your turn (your opponent stopped their clock) and you made your move in 2s and stopped your clock, you would still have 03:19 on your clock. If you had made your move in 6s, then your clock would have started decrementing 5 seconds into your turn and would have read 03:18 when you stopped it.

Unlike increment, you don't get to keep the time you save by moving quickly ... so your clock time never gets larger (as it can with increment time controls).

This large Continental Chess Association tournament uses a delay instead of increment. [14th Annual Southest Class Championships](https://new.uschess.org/14th-annual-southwest-class-championships). They describe the time control as "40/80, SD30, d30", meaning 80 minutes for the first 40 moves and 30 minutes for the remainder of the game with a delay of 30 seconds per move. I would encode that as \[TimeControl "40/4800d30:1800d30"\]
